src/tbb/governor.h

```diff
@@ -151,6 +158,68 @@ class governor {
         return false;
 #endif
     }
+
+#if __TBB_STATISTICS
+    enum class statistics : std::size_t {
+        exec_task = 0,
+        get_task,
+        task_res_not_null,
+        steal_task,
+        try_steal_task,
+        victim_pool_empty,
+        size,
+    };
+    using statistic_array = std::array<std::uint32_t, static_cast<std::size_t>(statistics::size)>;
+
+    class collect_statistics : no_copy {
+    public:
+        collect_statistics() = default;
+
+        ~collect_statistics() = default;
+
+        void emplace(array_stat thread_stat) {
+            std::lock_guard<std::mutex> lk {m_};
+            accumulate_statistics.emplace_back(thread_stat);
+        }
+
+        // thread-unsafe method
+        void print_statistics() {
+                int offset = 2;
+                /* std::string not so bad because of Small String Optimization */
+                std::string column_names[] = {
+                    "executed", "getted",
+                    "result != null", "steal", "try steal",
+                    "vic pool empty",
```

Comment:
Consider more verbose names and making this string static

Reply:
Names should be as short a possible, otherwise the table with be wider then the screen..